content/windows-stemcell-create.md

```
@@ -7,13 +7,13 @@ This topic describes how to create a Windows Stemcell for vSphere.
 To create a BOSH stemcell for Windows on vSphere, do the following:
 
 1. [Create a Base VM for the BOSH Stemcell](#create-base-vm)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)